includes/admin/class-wcmypa-admin.php

```diff
@@ -906,6 +918,85 @@ private function printDeliveryDate(DeliveryOptions $deliveryOptions): void
     }
 
     /**
+     * Output the delivery date if there is a date and the show delivery day setting is enabled.
+     *
+     * @param DeliveryOptions $deliveryOptions
+     * @param bool            $mail
+     *
+     * @throws \Exception
+     */
+    private function getConfirmationData(DeliveryOptions $deliveryOptions, bool $mail): void
```

Comment:
Kun je een duidelijkere naam bedenken voor $mail? (ook meteen in die andere functie)

Reply:
done